pkg/plugins/runtime/k8s/webhooks/gateway_instance_validator.go

```diff
@@ -49,6 +55,11 @@ func (h *GatewayInstanceValidator) ValidateDelete(ctx context.Context, req admis
 }
 
 func (h *GatewayInstanceValidator) ValidateCreate(ctx context.Context, req admission.Request) admission.Response {
+	if h.cpMode == config_core.Global {
+		return admission.Denied(fmt.Sprintf("Operation not allowed. %s resources like %s can be created only from the %s control plane and not from a %s control plane.",
+			version.Product, "MeshGatewayInstance", strings.ToUpper(config_core.Zone), strings.ToUpper(h.cpMode)))
```

Comment:
```suggestion
			version.Product, "MeshGatewayInstance", config_core.Zone, h.cpMode))
```

You can emphasise with quotes instead